### DISCUSSION TITLE
How to measure to ratio between small and big changes

### DISCUSSION THREAD

**Discussion by lidortal** [community/community#21893] [New to GitHub]
<p>Hi,</p>
<p>I want to have some statiscis on my repository.</p>
<p>One of the thing I want to check is if we work in small changes.</p>
<p>How can I get the ratio between small and big changes?</p>
<p>In addition is there a way to enforce small changes?</p>
<p>Thanks,</p>
<p>Lidor</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>There’s nothing built-in to GitHub to enforce things like what you’re requesting. There may be add-on tools that could do something like that. You might want to take a look at <a href="https://probot.github.io" rel="nofollow noopener">Probot</a> to build something like that. You could take a look at <a href="https://probot.github.io/apps/wip/" rel="nofollow noopener">WIP</a> for an example.</p>

**Comment by lidortal** [comment]
<p>Hi,</p>
<p>Can I install those applications on Github Enterprise 2.12? If so, where can I find the relevant documentation?</p>
<p>Thanks,</p>
<p>Lidor</p>

**Comment by lee-dohm** [comment]
<p>All of the Probot documentation is available at <a href="https://probot.github.io" rel="nofollow noopener">https://probot.github.io</a> (as mentioned in the README at <a href="https://github.com/probot/probot" rel="nofollow noopener">https://github.com/probot/probot</a>). If I recall correctly, Probot apps can be installed on the more recent versions of GitHub Enterprise but the <a href="https://github.com/probot/probot/issues" rel="nofollow noopener">Probot issues list</a> is probably the best place to find the latest information on that.</p>
